common/flatpak-installation.c

```diff
@@ -2663,3 +2663,30 @@ flatpak_installation_prune_local_repo (FlatpakInstallation *self,
 
   return flatpak_dir_prune (dir, cancellable, error);
 }
+
+/**
+ * flatpak_installation_run_triggers
+ * @self: a #FlatpakInstallation
+ * @cancellable: (nullable): a #GCancellable
+ * @error: return location for a #GError
+ *
+ * Run the trigger commands to update the files exported by the apps in
+ * @self. Should be used after one or more app install, upgrade or
+ * uninstall operations with the NO_TRIGGERS flag set.
```

Comment:
Might be nice to link to the exact flag names, and/or link to this function from the docs for each `NO_TRIGGERS` flag